core/rootcling_stage1/CMakeLists.txt

```diff
@@ -37,5 +35,12 @@ ROOT_EXECUTABLE(rootcling_stage1 src/rootcling_stage1.cxx
                               $<TARGET_OBJECTS:Foundation_Stage1>
                               LIBRARIES ${CLING_LIBRARIES} ${LINK_LIBS} ${CMAKE_DL_LIBS} ${CMAKE_THREAD_LIBS_INIT}
                               NOINSTALL)
+
+target_include_directories(rootcling_stage1 PRIVATE
+   ${CMAKE_SOURCE_DIR}/core/base/inc    # for RConfig.hxx
+   ${CMAKE_SOURCE_DIR}/core/dictgen/res # for rootcling_impl.h
+   ${CMAKE_BINARY_DIR}/ginclude         # for RConfigure.h
```

Comment:
@Axel-Naumann I see that we need to prioritize PR.... I tried to build ROOT master (without this PR) from scratch and I see next failure:
`[ 41%] Building CXX object core/rootcling_stage1/CMakeFiles/rootcling_stage1.dir/src/rootcling_stage1.cxx.o
cd /home/oksana/CERN_sources/root-rtcxxmodules/builds/core/rootcling_stage1 && /usr/bin/c++   -I/home/oksana/CERN_sources/root-rtcxxmodules/builds/include -I/home/oksana/CERN_sources/root-rtcxxmodules/core/dictgen/res  -std=c++11 -Wno-implicit-fallthrough -Wno-noexcept-type -pipe  -Wshadow -Wall -W -Woverloaded-virtual -fsigned-char -pthread -O3 -DNDEBUG   -std=c++11 -fno-rtti -o CMakeFiles/rootcling_stage1.dir/src/rootcling_stage1.cxx.o -c /home/oksana/CERN_sources/root-rtcxxmodules/core/rootcling_stage1/src/rootcling_stage1.cxx
/home/oksana/CERN_sources/root-rtcxxmodules/core/rootcling_stage1/src/rootcling_stage1.cxx:12:10: fatal error: RConfigure.h: No such file or directory
   12 | #include "RConfigure.h" `

Reply:
This PR will solve a current problem...